app/models/concerns/geocodeable.rb

```diff
@@ -3,6 +3,13 @@
 module Geocodeable
   extend ActiveSupport::Concern
 
+  # Does the given object have enough location info to set location info on a
```

Comment:
Why are there objects that don't give enough information, but still have a location?

Couldn't this just check if the lat & long are present? And  then be generalized, e.g. `has_location?` - which would be the same method that is checked to determine whether to geocode in the first place?

Reply:
> Why are there objects that don't give enough information, but still have a location?

Dirty data and/or different requirements for different objects, I imagine. 

For `Bike` records specifically, we expect a country and city or country and zipcode:

```rb
    object&.country.present? &&
      (object&.city.present? || object&.zipcode.present?)
```

A given accompanying record we're pulling location info from (stolen record, organization, etc) to set a bike's location may not satisfy that condition.

We're using more than just the Bike latitude and longitude (city, country, etc) elsewhere.